### PR TITLE
Add spider for Ananindeua/PA

### PR DESCRIPTION
**Pull request by VGasparini**
Hello :wave: 

This PR add spider for Ananindeua - Para

### CODE REVIEW

File: processing/data_collection/gazette/spiders/pa_ananindeua.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+from scrapy import Request
+from gazette.spiders.base import BaseGazetteSpider
+from gazette.items import Gazette
+from datetime import datetime
+import dateparser
```
Comment (jvanz) [inline processing/data_collection/gazette/spiders/pa_ananindeua.py:5]:
```suggestion
from datetime import datetime

import dateparser
from scrapy import Request

from gazette.spiders.base import BaseGazetteSpider
from gazette.items import Gazette
```

File: processing/data_collection/gazette/spiders/pa_ananindeua.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            range(1, avaliable_years_ids + offset), range(1, 13)
+        ):
+            yield Request(
+                url=f"{response.url}?id={year_id}&mes={month_id}",
+                callback=self.parse_month,
+            )
+
+    def parse_month(self, response):
+        file_urls = response.css(
+            "div#content div div#online_arquivo a::attr(href)"
```
Comment (arianerocha) [inline processing/data_collection/gazette/spiders/pa_ananindeua.py:30]: **suggestion**: change this line to a constant

`FILE_ELEMENT_CSS = "div#content div div#online_arquivo a::attr(href)"`

File: processing/data_collection/gazette/spiders/pa_ananindeua.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        ).extract()
+        dates = response.css("div#content div div#online_data::text").extract()
+
+        for date_str, file_url in zip(dates, file_urls):
+            date = dateparser.parse(date_str, date_formats=["%d/%m/%Y"]).date()
```
Comment (arianerocha) [inline processing/data_collection/gazette/spiders/pa_ananindeua.py:35]: suggestion: set a constant to date format

**Review by arianerocha — CHANGES_REQUESTED** [review]

**Review by jvanz — CHANGES_REQUESTED** [review]

File: processing/data_collection/gazette/spiders/pa_ananindeua.py (hunk 2 above)
Comment (VGasparini) [reply processing/data_collection/gazette/spiders/pa_ananindeua.py]: Done (:

File: processing/data_collection/gazette/spiders/pa_ananindeua.py (hunk 3 above)
Comment (VGasparini) [reply processing/data_collection/gazette/spiders/pa_ananindeua.py]: Done (:

File: processing/data_collection/gazette/spiders/pa_ananindeua.py (hunk 1 above)
Comment (VGasparini) [reply processing/data_collection/gazette/spiders/pa_ananindeua.py]: Done (:

**Review by gabubellon — APPROVED** [review]

**Review by jvanz — APPROVED** [review]

**Pull request merged**